packages/imperative/src/imperative/src/plugins/PluginManagementFacility.ts

```diff
@@ -937,7 +932,7 @@ export class PluginManagementFacility {
      *
      * Both range strings come from the package.json files of the plugin and the
      * hosting CLI. We consider the version ranges to be compatible if the two
```

Comment:
Thoughts on changing line 934 to "We consider the versions to be compatible if they satisfy the CLI version range." ?